en/sim_gazebo_gz/README.md

```diff
@@ -192,15 +223,27 @@ Here are some examples of the different scenarios covered above.
    PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_NAME=x500 ./build/px4_sitl_default/bin/px4
    ```
 
+4. **Start simulator in standalone mode + connect to Gazebo instance running default world**
+
+   ```sh
+   PX4_GZ_STANDALONE=1 PX4_SYS_AUTOSTART=4001 PX4_SIM_MODEL=gz_x500 ./build/px4_sitl_default/bin/px4
+   ```
+
+   In a separate terminal run:
+
+   ```sh
+   python simulation-gazebo --world default
```

Comment:
How is this in my path?

Reply:
Do I need to enter  `--world default`?

Reply:
My understanding is that PX4 would get that it is using a particular vehicle from `PX4_SYS_AUTOSTART=4001` and Gazebo would determine the vehicle from  `PX4_SIM_MODEL=gz_x500`. IS that right?

Reply:
It is not your path. I will adjust this. You do not actually need to enter `--world default`. However, I thought it would be useful to show in an example. I can take it out if you want. 
On your third question: Depends. If you start PX4 with `make` then you only need to provide the vehicle and PX4 would infer the id from that (and gazebo would receive the name). If you start with the binary directly and set the env variables, then your description is correct.

Reply:
> You do not actually need to enter --world default. However, I thought it would be useful to show in an example. I can take it out if you want.

I would probably have kept this, or shown both :-)

Reply:
But no need to change it back.